code/modules/events/blob.dm

```diff
@@ -14,11 +14,10 @@
 	var/mob/C
 	if(candidates.len)
 		C = pick(candidates)
-		Blob = new /obj/effect/blob/core(T, 200)
+		Blob = new /obj/effect/blob/core(T, 200, null, 2, 0, 1)

```

Comment:
/proc/create_overmind in code/game/gamemodes/blob/blobs/core.dm returns early the first time it is called. That has to be changed. Otherwise, New() calls create_overmind(new_overmind) but that still does nothing because override_delay is not set. Additionally, I did not trust that in the few seconds around spawn, a Life() call wouldn't call create_overmind() again, resulting in a second overmind. The method that I am currently using completely blocks create_overmind from being called automatically, but still allows it to be called manually to put the right player into the blob.